eb_hooks.py

```diff
@@ -204,6 +204,19 @@ def parse_hook_openblas_relax_lapack_tests_num_errors(ec, eprefix):
         raise EasyBuildError("OpenBLAS-specific hook triggered for non-OpenBLAS easyconfig?!")
 
 
+def parse_hook_Pillow_SIMD_harcoded_paths(ec, eprefix):
```

Comment:
Rather than make this a hook, it's probably better to add the fix upstream in EasyBuild (as was done for plain `PILLOW` in https://github.com/easybuilders/easybuild-easyconfigs/pull/19267)

Reply:
PR created: https://github.com/easybuilders/easybuild-easyconfigs/pull/19996#

Reply:
I will change the PR and remove the added hook, better initiate a rebuild for all archs?